examples/phabricator/README.md

```diff
@@ -188,10 +188,7 @@ Use the file [`examples/phabricator/phabricator-service.json`](phabricator-servi
     "selector": {
       "name": "phabricator"
     },
-    "createExternalLoadBalancer": true,
-    "publicIPs": [
-      "107.178.210.6"
-    ]
+    "type": "LoadBalancer"
```

Comment:
I think we should add instructions on how to get the Load Balancer's IP address, like [this](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/guestbook-go#step-six-create-the-guestbook-service)